src/SFA.DAS.EmployerIncentives.Api.AcceptanceTests/Extensions/DapperExtensions.cs

```diff
@@ -11,16 +11,11 @@
 using TableAttribute = Dapper.Contrib.Extensions.TableAttribute;
 
 // ReSharper disable once CheckNamespace
-namespace SFA.DAS.EmployerIncentives.Api.AcceptanceTests
+namespace SFA.DAS.EmployerIncentives.Functions.TestHelpers
```

Comment:
why the namespace change?

Reply:
I moved it into the shared project but had a runtime version issue so moved it back and didn't update the namespace.  Will look into it.

Reply:
Put back